src/core/Table/TablePaginator.tsx

```diff
@@ -158,18 +157,20 @@ export const TablePaginator = (props: TablePaginatorProps) => {
 
   const pageButton = React.useCallback(
     (index: number, tabIndex = index === focusedIndex ? 0 : -1) => (
-      <Button
-        key={index}
-        styleType='borderless'
-        className={cx({ 'iui-active': index === currentPage })}
-        onClick={() => onPageChange(index)}
-        aria-current={index === currentPage}
-        aria-label={localization.goToPageLabel(index + 1)}
-        tabIndex={tabIndex}
-        size={buttonSize}
-      >
-        {index + 1}
-      </Button>
+      <div key={index}>
+        <button
+          className={cx('iui-paginator-page-button', {
+            'iui-active': index === currentPage,
+            'iui-paginator-page-button-small': buttonSize === 'small',
```

Comment:
Why modifier class has whole class name in the beginning? DatePicker classes were refactored a bit but modifiers remained the same.

Reply:
This happens because of `&-`. I don't think it's a bad thing

Reply:
Yeah, I understand why this class is like this. What I don't understand is the reason why it was done like this and only here. It looks ugly to me or then it should be enough to have only `iui-paginator-page-button-small` without `iui-paginator-page-button`.

Reply:
then you won't be able to target base button with one class.

in fact this is a popular BEM convention
https://en.bem.info/methodology/naming-convention/#element-modifier-name

Reply:
But then we are inconsistent here. Everywhere we use short modifiers and only here (for whatever reason) we decided to go with BEM convention.

Reply:
How would you solve this? Inconsistencies are everywhere in the CSS repo, but this solution keeps the specificity low while avoiding conflicts with other components.

Also maybe not the right place for this discussion as we can't change the CSS in this PR.

Reply:
What is the problem with `.iui-paginator-page-button.iui-small`? If we don't have modifier classes as global class, we shouldn't have a problem with that.